plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -89,6 +90,7 @@ public class BigQueryClient
     private static final int PAGE_SIZE = 100;
 
     static final Set<TableDefinition.Type> TABLE_TYPES = ImmutableSet.of(TABLE, VIEW, MATERIALIZED_VIEW, EXTERNAL, SNAPSHOT);
+    private static final String TABLE_TYPES_QUERY_SET = TABLE_TYPES.stream().map(Object::toString).map(type -> "'" + type + "'").collect(joining(","));
```

Comment:
This conversion looks like the cause of CI failure. For instance, `table_type` column for normal tables should be `BASE TABLE` instead of `TABLE`.

Reply:
I will tomorrow morning my time, maybe I'll still catch you to retry CI tests

Reply:
This is weird because `BASE TABLE` do not exists in google's model `TableDefinition.Type`, there is only `TABLE` there

I added `BASE TABLE` as `String` constant in client, and added appropriate comment
also included that in query's predicate allowed values so it should work as expected now